src/tbb/arena_slot.cpp

```diff
@@ -125,6 +134,9 @@ d1::task* arena_slot::get_task(execution_data_ext& ed, isolation_type isolation)
                 tail.store(T0, std::memory_order_relaxed);
                 // The release fence is used in publish_task_pool.
                 publish_task_pool();
+                thread_data* td = ed.task_disp->m_thread_data;
+                td->my_arena->pool_mask[this_task_arena::current_thread_index()].store(
```

Comment:
```suggestion
                td->my_arena->pool_mask[td->my_arena_slot].store(
```
Can we use `td->my_arena_slot` instead of getting access to TLS via `this_task_arena::current_thread_index()`?